### ISSUE TITLE
Need little Help to implement something small

### ISSUE THREAD

**Issue by solisinvictum** [LinusDierheimer/fastfetch#353]
Hey @ all,

i gaved the source a little look. i thought i could implement something easy, because i know a little bit C (last time used in School 20 years ago lol).

There is a nice Distro, called CachyOS. One of the benefits is, thats automatic gets detected if the used Hardware is capable to use a specific, optimized Repo (actually -v3 called). In there are Packages wich are optimized for specific CPU Flags/Feature to improve Speed, loadingtime, etc pp.

If the Hardware support that, this repo gets enabled (but without the notification for the user, or a Info or somewhat similar).

I already "created" something like that, with few Lines of bash/fish. See here:

![grafik](https://user-images.githubusercontent.com/94938946/208847744-32131cf4-8c2e-47d0-b560-9fa4acfe9d7f.png)
![grafik](https://user-images.githubusercontent.com/94938946/208847797-424bc632-e04c-4c40-b59b-f3d05beb6126.png)

But instead of figure out how to format it correctly so it looks uniform (pipes, cat, sed, what_ever). i thought  i could create a plugin/module/edit_somewhere_in_the_source_directly (my first try was simply to pipe the output through a script, wich replace the "Packages:" entry with "Packages: 1234 (v3 Repo enabled)". Wich works, but isnt so "beauty" and a quick n dirty solution).

But to be honest, in my head this sounded easier as thought, because i dont know where to start in the source.

Could somebody give me a example, or a hint how to realize this?

Thanks very much!

**Comment by LinusDierheimer** [comment]
Take a look [here](https://github.com/LinusDierheimer/fastfetch/blob/dev/src/detection/packages/packages_linux.c). Note that pacman on Manjaro already gets some info appended, you can implement yours the same way.

**Comment by solisinvictum** [comment]
So Manjaro forked fastfetch? or what do you mean?

Thanks for the link. I give it a look and try it to understand :D

**Comment by LinusDierheimer** [comment]
No, on manjaro the output is either `(pacman)[stable]` or `(pacman)[unstable]` depending on the enabled repo. I suppose you want to do something similar, so i figured the code for that might help you.
